### DISCUSSION TITLE
Notifikasi Sistem PPDB

### DISCUSSION THREAD

**Discussion by fahram** [kirimwa-id/forum#58] [Pamerkan]
> Setelah testing beberapa hari, ternyata kirimwa.id ini lebih bagus daripada yang berbayar. terutama fitur webhook send_message_response. 

Sebetulnya aplikasi yang saya buat sudah pakai service wa lain yang berbayar, cuma sering disconnect dan harus scan manual lagi. Jadi kadang banyak pesan yang tidak terkirim.

### Fitur notifikasi whatsapp yang saya gunakan di sistem ppd :

- Notifikasi Nominal pembayaran dengan kode unik.
- Notifikasi Jadwal test
- Notifikasi Jadwal Pengumuman.
- Notifikasi User account (untuk kelengkapan data ppdb)
- Pesan Manual (custom)

Dan setelah saya cek ulang, ternyata pesan yang dikirim perhari ada yang lebih **400**. semoga bisa ditambahkan kuota hariannya. 😁 
![Screen Shot 2021-10-29 at 07 04 32](https://user-images.githubusercontent.com/11632202/139352794-04a1afe4-37b7-476b-a34f-9d70aef53438.png)

**Comment by rioastamal** [comment]
Terima kasih mas @fahram atas testimoni penggunaannya. Kalau perlu quota tambahan bisa request lagi.
